src/main/java/org/folio/rest/impl/ScheduledNoticesAPI.java

```diff
@@ -118,7 +118,7 @@ public void putScheduledNoticeStorageScheduledNoticesByScheduledNoticeId(String
   private Future<Void> executeSql(PostgresClient pgClient, String sql) {
 
     Promise<RowSet<Row>> promise = Promise.promise();
-    pgClient.execute(sql, promise.future());
+    pgClient.execute(sql, promise);
     return promise.future().map(ur -> null);
```

Comment:
The body of executeSql can be replaced by

    return pgClient.execute(sql).mapEmpty();

Hoiwever, as the return value of executeSql is not used the method executeSql should be deleted and deleteScheduledNoticeStorageScheduledNotices should directly call

    pgClient.execute(sql)

Reply:
Good point, thank you Julian.  Will fix this.